lab-dustin/lib/readfile.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { readFile } = require('fs');
+
+exports.readData = function(file, callback) {
+  readFile(`${__dirname}/../assets/${file}`, function(err, buff){
+    if (err) { callback(err); }
```

Comment:
You need to `return` here, either after `callback(err)` or all at once (`return callback(err);`). This fix would be a good thing to test-drive.